test/parallel/test-fs-utils-get-dirents.js

```diff
@@ -65,6 +67,19 @@ const filename = 'foo';
     },
     ));
 }
+{
+  // Intentional error in lstat
+  getDirents(
+    Buffer.from(nonExistentPath),
+    [[Buffer.from(filename)], [UV_DIRENT_UNKNOWN]],
+    common.mustCall((err) => {
+      assert.match(
+        err.code,
+        new RegExp(`^${'ENOTDIR'}|${'ENOENT'}$`)
```

Comment:
Why are these a template literal? Can't you just use a RegEx value?

Reply:
simplified to
```js
    common.mustCall((err) => {
      assert.match(
        err.code, /ENOTDIR|ENOENT/
      );
```